entities/ApiResult.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Routelandia\Entities;
+
+class ApiResult {
+	function __construct($input, $debug = null) {
+		is_array($input) ? $this->results = $input : $this->results = array($input);
+		is_null($debug) ? : $this->debug = $debug;
```

Comment:
This line was a little hard to read... 
I might have gone with something more like
```php
if(!is_null($debug) { $this->debug = $debug }
```